Aptoide/appcoins-wallet-android
# [MOB-880] Perks transactions

**What does this PR do?**

Adds the perks on the transaction list and sends a notification if a transaction results in a perk

**Database changed?**

   Yes (Added Perk, Subtype, title and description to the TransactionEntity table. It also deletes all entries from September 14th onwards so that users that update after that can have their perk transactions update to the new layout) 

**Where should the reviewer start?**

PerkBonusService.kt
Transaction.java
PerkBonusViewHolder

**How should this be manually tested?**

Level up and check that you received a notification with the bonus earned and check if the transaction appears with the new layout in the transactions screen.
Guarantee that when you move from an old version to this that nothing breaks due to the migration

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-880

**Code Review Checklist**
